etcdmain/etcd.conf.sample.yml

```diff
@@ -0,0 +1,135 @@
+# This is the configuration file for the etcd server.
+
+# Human-readable name for this member.
+name: 
```

Comment:
can we fill in all fields in the sample file rather than keep some of them empty? (probably as same as the flag default values)